core/src/main/res/values/strings.xml

```diff
@@ -402,4 +402,9 @@
   <string name="donation_dialog_title">Donate Today</string>
   <string name="donation_dialog_description">%s needs your help.</string>
   <string name="make_donation">Make a donation</string>
+  <string name="fetch_notification_download_complete">Complete</string>
```

Comment:
@amanna13 We should give our name to these strings instead of `fetch`, and it could lead to unexpected behavior when accessing it at runtime. See in the below image we are using this string, and the `R` class we have imported in this class(DownloadMonitorService) from the `fetch` so this string will not take ours. So to avoid confusion we should rename it. Additionally, there are already some strings available e.g. `Complete`, `Paused` we should use the existing ones.

![image](https://github.com/user-attachments/assets/313cb73c-9896-47f7-ab65-bee5c38b497e)